spec/integration/integration_helper.rb

```diff
@@ -5,6 +5,7 @@
 begin
   require 'active_support'
 rescue LoadError
+  print 'No "active_support" loaded'
```

Comment:
Support for DM has been dropped on master, can you remove this rescue and rebase the branch?

thanks